examples/index.html

```diff
@@ -199,16 +199,16 @@ <h2 style="margin-top:20px">Save</h2>
       <div class="col-md-6">
         <h2>Need Help?</h2>
         <p>
-          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues">PublicLab.org</a>
+          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues/new">PublicLab.org</a>
```

Comment:
```suggestion
          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues/new">Gitter</a>
```

Reply:
Hey why did you make it gitter??

Reply:
All the links being the same makes no sense